nifi-registry/nifi-registry-core/nifi-registry-framework/src/main/java/org/apache/nifi/registry/event/EventFactory.java

```diff
@@ -37,6 +37,10 @@ public static Event bucketCreated(final Bucket bucket) {
         return new StandardEvent.Builder()
                 .eventType(EventType.CREATE_BUCKET)
                 .addField(EventFieldName.BUCKET_ID, bucket.getIdentifier())
+                .addField(EventFieldName.BUCKET_NAME, bucket.getName())
+                .addField(EventFieldName.BUCKET_DESCRIPTION, bucket.getDescription())
```

Comment:
The description of a bucket is optional and can be null. Currently the event class doesn't accept null values so I think we'll have to make this conditional to only set the description if not null, or maybe set empty string in the event if null.

Running the integration tests for registry produces a failing test with:
```
[qtp1243712710-92] ERROR org.apache.nifi.registry.web.mapper.ThrowableMapper - An unexpected error. Returning Internal Server Error response.
java.lang.NullPointerException
	at java.base/java.util.Objects.requireNonNull(Objects.java:233)
	at org.apache.nifi.registry.event.StandardEventField.<init>(StandardEventField.java:35)
	at org.apache.nifi.registry.event.StandardEvent$Builder.addField(StandardEvent.java:96)
	at org.apache.nifi.registry.event.EventFactory.bucketCreated(EventFactory.java:41)
	at org.apache.nifi.registry.web.api.BucketResource.createBucket(BucketResource.java:94)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at **java.base/java.lang.reflect.Method.invoke(Method.java:580)**
```
From:
```
[ERROR] Errors:
[ERROR]   DBFlowStorageIT.testAll:94 » NiFiRegistry Error creating bucket: An unexpected error has occurred. Please check the logs for additional details.
```

`mvn clean install -Pintegration-tests` from `nifi/nifi-registry`